source/MaterialXTest/GenShaderUtil.cpp

```diff
@@ -17,703 +17,730 @@ namespace mx = MaterialX;
 namespace GenShaderUtil
 {
 
-bool getShaderSource(mx::GenContext& context,
-                    const mx::ImplementationPtr implementation,
-                    mx::FilePath& sourcePath,
-                    mx::FilePath& resolvedPath,
-                    std::string& sourceContents)
-{
-    if (implementation)
+    bool getShaderSource(mx::GenContext& context,
```

Comment:
Hard to tell what changed here. Can you perform the un-indentation so can do the compare? Thanks.